### PR TITLE
Closing panels with ESC key

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
Hi,

In this PR i added event handlers for escape button (keyboard) to close panels.

https://github.com/mitodl/lore/issues/483
@pdpinch @carsongee @noisecapella

### CODE REVIEW

**Comment by noisecapella** [comment]
This doesn't work on Firefox

**Comment by noisecapella** [comment]
It seems that there should be only one key handler and it should close whatever panel happens to be open at the time. You should have only one event handler for this. The `keyup` function will attach event handlers without removing any so on repeated panel opens you will have multiple event handlers being called for every key press.

**Comment by amir-qayyum-khan** [comment]
@noisecapella it is working on firefox on mac.

**Comment by amir-qayyum-khan** [comment]
@noisecapella centralised code to avoid multiple event binding.

File: ui/static/ui/js/listing.js [hunk 1]
```diff
@@ -80,6 +80,23 @@ define('listing',
 
       $('[data-toggle=popover]').popover();
 
+       $(document).keyup(function(e) {
```
Comment (noisecapella) [inline ui/static/ui/js/listing.js]: Please add a comment before this block saying 'Close panels on escape keypress'. You will also need to fix indentation here

**Comment by noisecapella** [comment]
This works for me now on Firefox, not sure why it wasn't working before.

**Comment by amir-qayyum-khan** [comment]
Done @noisecapella

**Comment by noisecapella** [comment]
Can you git amend and repush? Looks like a connection timeout on the tests, probably unrelated to this

**Comment by noisecapella** [comment]
I think this is ready to merge when tests pass

**Pull request merged**